src/structures/GuildBan.js

```diff
@@ -7,11 +7,6 @@ const Base = require('./Base');
  * @extends {Base}
  */
 class GuildBan extends Base {
-  /**
-   * @param {Client} client The instantiating client
-   * @param {APIBanData} data The data for the ban
```

Comment:
This was the last instance of `APIBanData`. Should we remove this too?

https://github.com/discordjs/discord.js/blob/395186da7241b1ea704e4d5e0c6df6fd4655eb5c/src/structures/GuildBan.js#L59-L62